src/main/java/core/basesyntax/file/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.file;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
+        try {
+            Files.write(Paths.get(filePath), data.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write file" + filePath, e);
```

Comment:
It's good that you're rethrowing the exception with an informative message, but the message is missing a space before the file path. It should be "Can't write to file " + filePath for better readability.